Libplanet/Net/Messages/Blocks.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using System.Linq;
+using NetMQ;
+
+namespace Libplanet.Net.Messages
+{
+    internal class Blocks : Message
+    {
+        public Blocks(IEnumerable<byte[]> payloads)
```

Comment:
As a `Blocks` message cannot have more than 2<sup>32</sup> blocks, it would be better to check the number of `payloads`.